packages/host/tests/acceptance/code-mode-test.ts

```diff
@@ -1643,8 +1644,11 @@ module('Acceptance | code mode tests', function (hooks) {
       )
       .hasText('Link');
 
-    assert.ok(
-      getMonacoContent().includes('favPerson = linksTo(() => Person);'),
+    await waitFor('[data-test-editor]');
+
+    await waitUntil(
+      () => getMonacoContent().includes('favPerson = linksTo(() => Person);'),
+      { timeout: 2000 },
```

Comment:
Actually not sure if such a big timeout is needed but it seemed to have helped on my slowish computer